src/commands/dev/socket/client.rs

```diff
@@ -0,0 +1,84 @@
+use chrome_devtools::events::DevtoolsEvent;
+use console::style;
+
+use openssl::ssl::{SslConnector, SslMethod, SslStream, SslVerifyMode};
+
+use ws::util::TcpStream;
+use ws::{Handler, Handshake, Message as WsMessage, Sender};
+
+use url::Url;
+
+pub struct WsClient {
+    pub out: Sender,
+}
+
+impl Handler for WsClient {
+    fn on_open(&mut self, _: Handshake) -> ws::Result<()> {
+        let enable_profiler = WsMessage::text(
+            r#"{
+                "id": 1,
+                "method": "Profiler.enable"
+            }"#,
+        );
+        let enable_runtime = WsMessage::text(
+            r#"{
+                "id": 2,
+                "method": "Runtime.enable"
+            }"#,
+        );
+        let enable_debugger = WsMessage::text(
+            r#"{
+                "id": 3,
+                "method": "Debugger.enable"
+            }"#,
+        );
```

Comment:
they're pretty well contained now, but if we end up generating any more of these outside this function, i would suggest some structs w/ serializers rather than formatted strings.

Reply:
yes these will eventually be moved to the devtools crate

Reply:
Agreed, these should be structs whose "method" strings are consts.